multiple-matches.suite/Scripts/spitta.script

```diff
@@ -0,0 +1,9 @@
+﻿
```

Comment:
You should really have a connection process here for the screenshot. These suites and then solutions should be self-contained.